### PR TITLE
Added error handling for unexpected responses

### LINKED ISSUE #1155: Unexpected Response: already_claimed
https://app.getsentry.com/gostellarorg/stellar-client/group/52838508/

### LINKED ISSUE #1156: Unexpected Response: facebook_error
https://app.getsentry.com/gostellarorg/stellar-client/group/52835962/

### PR DESCRIPTION
**Pull request by bartekn**
Close #1155.
Close #1156.

PS. Removed duplicate `facebook_error` case.

### CODE REVIEW

File: app/scripts/controllers/facebook-reward-controller.js [hunk 1]
```diff
@@ -266,6 +258,9 @@ sc.controller('FacebookRewardCtrl', function ($rootScope, $scope, $http, $q, $an
         case 'already_taken':
           $scope.reward.updateReward('already_taken');
           break;
+        case 'already_claimed':
```
Comment (nullstyle) [inline app/scripts/controllers/facebook-reward-controller.js]: This is redundant with the fall through case below in this same patch.

Comment (bartekn) [inline app/scripts/controllers/facebook-reward-controller.js]: `case 'already_claimed':` below is in a different function.

Comment (nullstyle) [inline app/scripts/controllers/facebook-reward-controller.js]: my bad.

**Comment by nullstyle** [comment]
:+1:

**Pull request merged**